sycl/unittests/kernel-and-program/PersistentDeviceCodeCache.cpp

```diff
@@ -245,21 +251,23 @@ TEST_F(PersistenDeviceCodeCache, CorruptedCacheFiles) {
   // Only source file is present
   detail::PersistentDeviceCodeCache::putItemToDisc(Dev, Img, {}, BuildOptions,
                                                    NativeProg);
-  assert(!llvm::sys::fs::remove(ItemDir + "/0.bin") &&
-         "Failed to remove binary file");
+  EXPECT_EQ(!llvm::sys::fs::remove(ItemDir + "/0.bin"), true)
```

Comment:
```suggestion
  EXPECT_FALSE(llvm::sys::fs::remove(ItemDir + "/0.bin"))
```
https://google.github.io/googletest/reference/assertions.html#boolean